packages/react/src/components/Loading/Loading.stories.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Loading from '.';
+import mdx from './Loading.mdx';
+
+export default {
+  title: 'Components/Loading',
+  component: Loading,
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const Default = (args) => {
```

Comment:
Can we have a `Default` story that does not reference the `args` and instead uses the props directly so that users can copy and paste the story? The `args` should only be spread into the `Playground` story

Reply:
will do!

Reply:
Ok, i made a default and a playground story. Spread the args into the Playground. Curious why the default story doesn't retain the styling that the playground has...